lib/BootTidal.hs

```diff
@@ -27,7 +27,7 @@ let only = (hush >>)
     nudgeAll = streamNudgeAll tidal
     all = streamAll tidal
     resetCycles = streamResetCycles tidal
---    setCycle = streamSetCycle tidal -- commenting this line. Generates error in tidal 1.9.2
+    setCycle = streamSetCycle tidal -- commenting this line. Generates error in tidal 1.9.2
```

Comment:
The comment can be removed as well, at least the "commenting this line" part

Reply:
Done. I forgot about that!